testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -3206,6 +3206,23 @@ QUnit.test("adding the custom tag should clear input value (T385448)", function(
     assert.equal($input.val(), "", "the input is empty");
 });
 
+QUnit.test("adding the custom tag shouldn't lead to duplicating of ordinary tags ", function(assert) {
+    var $tagBox = $("#tagBox").dxTagBox({
+            acceptCustomValue: true,
+            items: [1, 2, 3]
+        }),
+        $input = $tagBox.find("input");
+
+    keyboardMock($input)
+        .type("custom")
+        .press("enter");
+
+    $($tagBox.find("." + LIST_ITEM_CLASS).first()).trigger("dxclick");
```

Comment:
Why we should generate `dxclick` here?

Reply:
We need to pick the ordinary tag from the list like [here](https://isc.devexpress.com/Thread/WorkplaceDetails/T690840)

Reply:
ok